javascript/core/js-history/infix-operators-are-left-associative.md

```diff
@@ -71,11 +50,11 @@ var x = a - (b - c);
 What does the following JS code output? ???
 
 ```javascript
-var a = 20;
-var b = 4;
-var c = 12;
+let a = 20;
+let b = 4;
+let c = 12;
 
-var x = a - (-b * c);
+let x = a - (-b * c);
 console.log(x);
 ```
 
```

Comment:
Revision question should be:

The processing of operations starts with the ??? operator if no brackets are used.

Missing a "the"

Reply:
I wasn't able to find how to put the comment on the revision question part, so I placed it as close as I could.